### PR TITLE
[doc]add readme

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Nebula Graph Studio
+
+**Nebula Graph Studio** 是一款可视化的图数据库 Web 应用，集构图、数据导入、图探索于一体，无缝连接图数据库，给用户带来全新的图数据库使用体验.
+
+## 数据导入
```
Comment (wilsonyou) [inline README-CN.md]: This is probably not accurate

Comment (Amber1990Zhang) [reply README-CN.md]: then what do you think the accurate expression is

File: README-CN.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+**Nebula Graph Studio** 支持从多种数据源导入数据。
+
+## 图探索
+
+相比文本格式的数据，可视化能够帮助更好地理解数据。导入数据后即可查看数据构图并对数据进行探索。
+
+## 文档
+
+- [User Manual](./docs/nebula-graph-studio-user-guide-en.md)
+- [用户手册](./docs/README-zh.md)
```
Comment (wilsonyou) [inline README-CN.md]: [用户手册](./docs/nebula-graph-studio-user-guide-cn.md)

Comment (wilsonyou) [reply README-CN.md]: (./docs/nebula-graph-studio-user-guide-en.md)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+You can import data from other sources into **Nebula Graph Studio**.
+
+## Graph Visualization
+
+Graph visualization provides a better understanding of your data than textual formats can support. When importing your data, you can see how your data looks and explore connections that you never know.
+
+## Documentation
+
+- [User Manual](./docs/nebula-graph-studio-user-guide-en.md)
+- [用户手册](./docs/README-zh.md)
```
Comment (wilsonyou) [inline README.md]: Chinese

File: README.md [hunk 4]
```diff
@@ -1,3 +1,24 @@
-# nebula-graph-studio
-- [English Mannual](./docs/nebula-graph-studio-user-guide-en.md)
-- [中文手册](./docs/nebula-graph-studio-user-guide-cn.md)
+# Nebula Graph Studio
+
+**Nebula Graph Studio** is a visualization application that combines data import and graph exploration.
+
+## Data Import
+
+You can import data from other sources into **Nebula Graph Studio**.
```
Comment (wilsonyou) [inline README.md]: Import data from your local computer to Nebula Graph

Comment (Amber1990Zhang) [reply README.md]: I think it's ok

File: README.md (hunk 3 above)
Comment (nianiaJR) [reply README.md]: this chines link is not right, change to ./docs/nebula-graph-studio-user-guide-zh.md

**Pull request merged**